cookbooks/Getting-Started/travel.aiconfig.json

```diff
@@ -6,6 +6,7 @@
     "models": {
       "gpt-3.5-turbo": {
         "model": "gpt-3.5-turbo",
+        "system_prompt": null,
```

Comment:
This should be removed, right? @rossdanlm

Reply:
I supposed you are correct